public/app/plugins/datasource/prometheus/specs/metric_find_query_specs.ts

```diff
@@ -43,8 +43,8 @@ describe('PrometheusMetricFindQuery', function() {
           {__name__: "metric", resource: "value3"}
         ]
       };
-      ctx.$httpBackend.expect('GET', 'proxied/api/v1/series?match[]=metric').respond(response);
-      var pm = new PrometheusMetricFindQuery(ctx.ds, 'label_values(metric, resource)');
+      ctx.$httpBackend.expect('GET', /proxied\/api\/v1\/series\?match\[\]=metric&start=.*&end=.*/).respond(response);
```

Comment:
Let's have one test which verified that the right start/end values are set? So that it's not possible to change the code to send a request with `...start=2016104` or so?